assets/js/modules/pagespeed-insights/components/common/Recommendations.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Recommendations component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME, STRATEGY_MOBILE, STRATEGY_DESKTOP } from '../../datastore/constants';
+import Recommendation from './Recommendation';
+const { useSelect } = Data;
+
+export default function Recommendations( { referenceURL, strategy } ) {
+	const finishedResolution = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getReport', [ referenceURL, strategy ] ) );
+	const recommendations = useSelect( ( select ) => {
+		const allAudits = select( STORE_NAME ).getAudits( referenceURL, strategy, true );
```

Comment:
See my other comment, this should rather call a separate `getAuditsWithStackPack( referenceURL, strategy, 'wordpress' )` selector.

Reply:
Updated.